app/models/school.rb

```diff
@@ -16,10 +16,12 @@
 #  index_schools_on_urn         (urn) UNIQUE
 #
 class School < ApplicationRecord
-  has_one :gias_school, foreign_key: :urn, primary_key: :urn
+  belongs_to :gias_school, foreign_key: :urn, primary_key: :urn
```

Comment:
This is how it should have been implemented when I merged in this code earlier in the week.